tools/ci_build/github/linux/docker/scripts/install_ubuntu.sh

```diff
@@ -111,7 +111,10 @@ fi
 
 if [ "$OS_VERSION" = "16.04" ]; then
     if [ "$PYTHON_VER" != "3.5" ]; then
+        # https://stackoverflow.com/questions/43621584/why-cant-i-install-python3-6-dev-on-ubuntu16-04
```

Comment:
This is not needed, `software-properties-common` already installed.

Reply:
I have undone my change to install_ubuntu.sh.